src/BrownBag/View.elm

```diff
@@ -105,21 +105,19 @@ shuffleView =
         ]
 
 
-
--- TODO: Remove confusing function identifier
-
-
-presentersList : List BrownBagPresenter -> Html msg
-presentersList brownBags =
+whosOnTheList : List BrownBagPresenter -> Html msg
+whosOnTheList brownBags =
     brownBags
```

Comment:
@waiyaki Could we change this variable to `undoneBrownbags` or something more expressive?